constants/currency.ts

```diff
@@ -79,7 +79,10 @@ export const sUSD_EXCHANGE_RATE = new Wei(1);
 export const SYNTH_DECIMALS = 18;
 
 export const ETH_ADDRESS = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'; // For 1inch API
-export const ETH_COINGECKO_ADDRESS = '0x4200000000000000000000000000000000000006'; // For coingecko API
+
+// For coingecko API
+export const ETH_COINGECKO_ADDRESS = '0x4200000000000000000000000000000000000006';
+export const KWENTA_COINGECKO_ADDRESS = '0x920cf626a271321c151d027030d5d08af699456b';
```

Comment:
Not sure 'COINGECKO' is needed, as it's just the contract address and nothing specific to coingecko I think `KWENTA_ADDRESS` makes more sense.  Not sure the background behind the ETH coingecko address above.

Reply:
Yes, you are right. There is no need to add `COINGECKO` for KWENTA address.
1inch API and Coingecko API use two ETH addresses in the token list.